apps/web/src/locales/en/messages.json

```
@@ -7312,6 +7312,15 @@
   "passkeyNotCopiedAlert": {
     "message": "The passkey will not be copied to the cloned item. Do you want to continue cloning this item?"
   },
+  "modifiedCollectionManagement": {
+    "message": "Modified collection management setting $ID$.",
+    "placeholders": {
+      "id": {
+        "content": "$1",
+        "example": "Limit Collection Creation/Deletion"
```

This is a hint to translators to tell them what will be inserted instead of `$ID$`. In this case I believe it's the orgId, not the name of the collection management setting. So:

```suggestion
        "example": "Unique ID"
```